### PR TITLE
Update minikube.md

### PR DESCRIPTION
**Pull request by Rajakavitha1**
- rewrite the Minikube instructions to set the PROXY_IP
- Addresses: https://github.com/Kong/docs.konghq.com/issues/5200
_ another related PR that addresses the command and output is https://github.com/Kong/docs.konghq.com/pull/5842

### Description

 
Include any supporting resources, e.g. link to a Jira ticket, GH issue, FTI, Slack, Aha, etc.

### Testing instructions

Tested and validated the instructions:
```
Last login: Tue Jul 25 14:28:04 on ttys001
rajakavithakodhandapani@Rajakavithas-MacBook-Pro-2 ~ % kubectl get -n kong services
NAME                      TYPE           CLUSTER-IP      EXTERNAL-IP   PORT(S)                      AGE
kong-admin                ClusterIP      None            <none>        8444/TCP                     74d
kong-proxy                LoadBalancer   10.98.71.100    127.0.0.1     80:31982/TCP,443:32722/TCP   74d
kong-validation-webhook   ClusterIP      10.106.75.254   <none>        443/TCP                      74d
rajakavithakodhandapani@Rajakavithas-MacBook-Pro-2 ~ % export PROXY_IP=127.0.0.1
rajakavithakodhandapani@Rajakavithas-MacBook-Pro-2 ~ % echo $PROXY_IP
127.0.0.1
rajakavithakodhandapani@Rajakavithas-MacBook-Pro-2 ~ % curl -i $PROXY_IP
HTTP/1.1 404 Not Found
Date: Tue, 25 Jul 2023 09:05:52 GMT
Content-Type: application/json; charset=utf-8
Connection: keep-alive
Content-Length: 52
X-Kong-Response-Latency: 11
Server: kong/3.3.0

{
  "message":"no Route matched with those values"
}%                                                                                                                   rajakavithakodhandapani@Rajakavithas-MacBook-Pro-2 ~ % >....                                                         
        ports:
        - containerPort: 1027
        env:
          - name: NODE_NAME
            valueFrom:
              fieldRef:
                fieldPath: spec.nodeName
          - name: POD_NAME
            valueFrom:
              fieldRef:
                fieldPath: metadata.name
          - name: POD_NAMESPACE
            valueFrom:
              fieldRef:
                fieldPath: metadata.namespace
          - name: POD_IP
            valueFrom:
              fieldRef:
                fieldPath: status.podIP
        resources: {}
" | kubectl apply -f -


service/echo unchanged
deployment.apps/echo unchanged
rajakavithakodhandapani@Rajakavithas-MacBook-Pro-2 ~ % echo "
apiVersion: networking.k8s.io/v1
kind: IngressClass
metadata:
  name: kong
spec:
  controller: ingress-controllers.konghq.com/kong
" | kubectl apply -f -


ingressclass.networking.k8s.io/kong unchanged
rajakavithakodhandapani@Rajakavithas-MacBook-Pro-2 ~ % echo "
apiVersion: networking.k8s.io/v1
kind: Ingress
metadata:
  name: echo
  annotations:
    konghq.com/strip-path: 'true'
spec:
  ingressClassName: kong
  rules:
  - host: kong.example
    http:
      paths:
      - path: /echo
        pathType: ImplementationSpecific
        backend:
          service:
            name: echo
            port:
              number: 1027
" | kubectl apply -f -


ingress.networking.k8s.io/echo unchanged
rajakavithakodhandapani@Rajakavithas-MacBook-Pro-2 ~ % curl -i http://kong.example/echo --resolve kong.example:80:$PROXY_IP


HTTP/1.1 200 OK
Content-Type: text/plain; charset=utf-8
Content-Length: 134
Connection: keep-alive
Date: Tue, 25 Jul 2023 09:07:40 GMT
X-Kong-Upstream-Latency: 72
X-Kong-Proxy-Latency: 111
Via: kong/3.3.0

Welcome, you are connected to node minikube.
Running on Pod echo-f4fdf987c-mls75.
In namespace default.
With IP address 10.244.0.31.
```

### Checklist 

- [ ] Review label added 
- [ ] PR pointed to correct branch (`main` for immediate publishing, or a release branch: e.g. `release/gateway-3.2`, `release/deck-1.17`)

### CODE REVIEW

File: app/_src/kubernetes-ingress-controller/deployment/minikube.md [hunk 1]
```diff
@@ -30,14 +30,21 @@ Next, we will setup an environment variable with the IP address at which
...
-# $ export PROXY_IP=$(minikube service <release-name>-kong-proxy --url | head -1)
-$ echo $PROXY_IP
+# minikube service <release-name>-kong-proxy --url | head -1
+```
+The output is similar to:
+```bash
 http://192.168.99.100:32728
 ```
-
+1. To set the environment variable, replace `<ip-address>` with the IP address at which Kong is accessible:
```
Comment (lena-larionova) [inline app/_src/kubernetes-ingress-controller/deployment/minikube.md:43]:
```suggestion
1. Set the environment variable. Replace `<ip-address>` with the {{site.base_gateway}} IP address you just retrieved:
```

File: app/_src/kubernetes-ingress-controller/deployment/minikube.md [hunk 2]
```diff
@@ -30,14 +30,21 @@ Next, we will setup an environment variable with the IP address at which
...
 
+1. Get the IP address at which Kong is accessible:
 ```bash
-$ export PROXY_IP=$(minikube service -n kong kong-proxy --url | head -1)
+$ minikube service -n kong kong-proxy --url | head -1
 # If installed by helm, service name would be "<release-name>-kong-proxy".
-# $ export PROXY_IP=$(minikube service <release-name>-kong-proxy --url | head -1)
-$ echo $PROXY_IP
+# minikube service <release-name>-kong-proxy --url | head -1
+```
```
Comment (lena-larionova) [inline app/_src/kubernetes-ingress-controller/deployment/minikube.md:38]: Can you please indent the codeblocks in both steps by three spaces?

**Review by lena-larionova — CHANGES_REQUESTED** [review]
Just a few minor edits.

File: app/_src/kubernetes-ingress-controller/deployment/minikube.md (hunk 2 above)
Comment (Rajakavitha1) [reply app/_src/kubernetes-ingress-controller/deployment/minikube.md]: Thank you for the review @lena-larionova . Just committed the changes.

**Review by lena-larionova — APPROVED** [review]
LGTM! Thanks for making this fix, and for applying all the feedback

**Pull request merged**